### PR TITLE
Add VPN support

### PR DESCRIPTION
**Pull request by voidrot**
Added an additional docker-compose and .env file to support VPN

This is related to #63

### CODE REVIEW

**Comment by lardbit** [comment]
Great, thanks.  I'll hopefully get to review this this week.

**Comment by lardbit** [comment]
I'm going to try to use what you put together and break apart the sections using a docker [extends](https://docs.docker.com/compose/extends/#multiple-compose-files) feature to avoid duplication of the compose files.  I think this scenario is a good fit for it.

**Comment by voidrot** [comment]
I agree I don't play with compose much so I didn't know that it had the capability to extend. if you want/need any help from me with it let me know.

**Comment by lardbit** [comment]
I don't think you can route jackett through the transmission-openvpn service.  I tested this to confirm and you can't access http://localhost:9117 to setup any indexers.

Can you explain how the jackett service is supposed to be routed through the transmission-openvpn service? I can't wrap my head around how that would work.

**Comment by voidrot** [comment]
the magic trick is in the the `network_mode` key that I missed adding to the jackett service, what it does is routes all the networking for jackett through the transmission/openvpn service and hence the exposed port on transmission.

File: docker-compose.vpn.yml [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+  # See Documentation for additonal environment options that may be needed per
+  # your vpn provider
+  # VPN Docs: https://haugene.github.io/docker-transmission-openvpn/
+  jackett:
+    labels:
+      - com.centurylinklabs.watchtower.enable=true
+    image: linuxserver/jackett
+    mem_limit: 200m
+    restart: always
+    network_mode: service:transmission
```
Comment (voidrot) [inline docker-compose.vpn.yml:73]: @lardbit this is the part that I missed that ties it together to route over the transmission container.

**Comment by lardbit** [comment]
Ah, that's pretty cool.  I didn't know you could do that with docker. It worked great.

I wasn't able to commit to your fork so I pulled your changes and made some updates to use docker's extend feature.  See https://github.com/lardbit/nefarious/pull/69 and let me know if you can test that on your end when you get some time.

**Pull request merged**